docs/research/rhubarb_test/build/index.js

```diff
@@ -0,0 +1,37 @@
+"use strict";
+Object.defineProperty(exports, "__esModule", { value: true });
+var node_child_process_1 = require("node:child_process");
+var node_process_1 = require("node:process");
+console.log("Start Rhubard....");
+console.log("Starting directory: $".concat((0, node_process_1.cwd)()));
+(0, node_process_1.cwd)();
+try {
```

Comment:
when we integrate it further later on, we'll probably want to save the current working directory before changing it. That way, we can go back to where we were afterwards.